migrate/migrations/20190613111312-add-schedule-favorites.sql

```diff
@@ -0,0 +1,22 @@
+-- +migrate Up
+ALTER TABLE user_favorites 
+ADD COLUMN tgt_schedule_id UUID 
+REFERENCES schedules(id);
+
+ALTER TABLE user_favorites 
+ALTER COLUMN tgt_service_id 
+DROP NOT NULL;
+
+ALTER TABLE user_favorites 
+ADD CONSTRAINT user_fav_schedules UNIQUE (user_id, tgt_schedule_id);
```

Comment:
I think we can combine these into one statement.

```sql
ALTER TABLE user_favorites
    ADD COLUMN tgt_schedule_id UUID REFERENCES schedules(id),
    ALTER COLUMN tgt_service_id DROP NOT NULL,
    ADD CONSTRAINT user_favorites_user_id_tgt_schedule_id_key UNIQUE(user_id, tgt_schedule_id);
```

As for the name I propose we go with the default, `user_favorites_user_id_tgt_schedule_id_key`. Postgres generates index names by `${tableName}_${columnName}_${otherColumnName}_key` since they have to be globally unique.